app/src/main/res/values/styles.xml

```diff
@@ -8,4 +9,36 @@
         <item name="colorAccent">@color/colorAccent</item>
     </style>
 
+    <!-- Base TextInputLayout style-->
+    <style name="TextInputLayout_Style" parent="Widget.Design.TextInputLayout">
+        <item name="android:layout_width">match_parent</item>
+        <item name="android:layout_height">60dp</item>
+        <!--<item name="layout_alignParentStart">true</item>-->
+    </style>
+
+    <!-- Base button style -->
+    <style name="AppButton_Style" parent="Base.Widget.AppCompat.Button">
```

Comment:
Apply the style to the entire application and do not add it to every element. Conversely, add only if it differs from the main style of the application.
Can be renamed to AppTheme.Button, AppTheme.EditText ...

https://stackoverflow.com/questions/10903647/set-a-consistent-style-to-all-edittext-for-e-g